keyboards/primekb/prime_e/keymaps/milestogo/keymap.c

```diff
@@ -0,0 +1,125 @@
+        {KC_TAB, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, KC_NO, KC_NO}, {B_2ME, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, KC_NO, K112, KC_NO, KC_NO}, {KC_LSFT, K201, K202, K203, K204, K205, KC_TRNS, K207, K208, K209, K210, K211, KC_TRNS, KC_NO, KC_NO}, {KC_LALT, KC_TRNS, KC_NO, KC_TRNS, KC_TRNS, KC_NO, KC_TRNS, KC_NO, KC_TRNS, KC_NO, KC_NO, KC_TRNS, KC_RSFT, KC_NO, KC_NO}, { KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, B_1ME, KC_ENTER } \
+    }
+
+#define LAYOUT_wrap_m2primee(...) LAYOUT_m2primee(__VA_ARGS__)
+#define LAYOUT_wrap_m2primee_trns(...) LAYOUT_m2primee_trns(__VA_ARGS__)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_QWERTY] = LAYOUT_wrap_m2primee(KC_GRAVE, _________________QWERTY_L1_________________, _________________QWERTY_R1_________________, KC_MINS, KC_EQL, KC_TAB, _________________QWERTY_L2_________________, _________________QWERTY_R2_________________, KC_QUOT, KC_LSFT, _________________QWERTY_L3_________________, MO(_SYM), _________________QWERTY_R3_________________, KC_RSFT, KC_LALT, MO(_MOV), LT(_NUM, KC_DEL), KC_BSPC, KC_SPC, LT(_NUM, KC_ESC), MO(_MOV), KC_RALT, KC_LGUI, KC_ENTER),
+
+    [_CDH] = LAYOUT_wrap_m2primee_trns(______________COLEMAK_MOD_DH_L1____________, ______________COLEMAK_MOD_DH_R1____________, KC_TRNS, KC_TRNS, ______________COLEMAK_MOD_DH_L2____________, ______________COLEMAK_MOD_DH_R2____________, KC_QUOT, ______________COLEMAK_MOD_DH_L3____________, KC_TRNS, ______________COLEMAK_MOD_DH_R3____________),
+
+    [_SYM] = LAYOUT_wrap_m2primee_trns(___________________SYM_L1__________________, ___________________SYM_R1__________________, KC_TRNS, KC_TRNS, ___________________SYM_L2__________________, ___________________SYM_R2__________________, KC_TRNS, ___________________SYM_L3__________________, KC_TRNS, ___________________SYM_R3__________________),
+
+    [_NUM] = LAYOUT_wrap_m2primee(KC_TRNS, __________40_______NUM_L1__________________, __________40_______NUM_R1__________________, KC_VOLD, KC_VOLU, KC_TRNS, __________40_______NUM_L2__________________, __________40_______NUM_R2__________________, KC_ENT, KC_TRNS, __________40_______NUM_L3__________________, KC_TRNS, __________40_______NUM_R3__________________, KC_TRNS, KC_TRNS, KC_TRNS, KC_LSFT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS),
+
+    [_MOV] = LAYOUT_wrap_m2primee_trns(__________40_______MOV_L1__________________, __________40_______MOV_R1__________________, KC_TRNS, KC_CDH, __________40_______MOV_L2__________________, __________40_______MOV_R2__________________, KC_TRNS, __________40_______MOV_L3__________________, KC_TRNS, __________40_______MOV_R3__________________),
+};
```

Comment:
You may want to wrap these to match the actual layout.  It looks like cformat removed any newlines. That can be prevented by putting `// clang-format off` before the code where you want the formatting preserved, and `// clang-format on` after it.